LICENSE

```diff
@@ -1,7 +1,7 @@
 
 The MIT License (MIT)
 
-Copyright (c) 2018 Kehittyvien Conien Suomi ry, Aarni Koskela
+Copyright (c) 2018-2021 Kehittyvien Conien Suomi ry, Aarni Koskela
```

Comment:
I guess no-one of us has actually assigned copyright to KcS, so we should probably just list individual names here.